Server/config/passport.config.js

```diff
@@ -1,13 +1,49 @@
+const passport = require('passport');
+var LocalStrategy = require('passport-local').Strategy;
+const passportJWT = require('passport-jwt');
+const JWTStrategy = passportJWT.Strategy;
+const ExtractJwt = require('passport-jwt').ExtractJwt;
+const bcrypt = require('bcrypt');
+const userModel = require('../models/user.model');
+const saltRounds = 10;
```

Comment:
What is use and significance of `saltRounds `

Reply:
saltrounds is a variable used in brcypt that defines the cost factor . More the cost factor , more the computation time